tel2puml/otel_to_pv/otel_to_pv.py

```diff
@@ -120,34 +123,57 @@ def handle_save_events(
             f"Error creating directory {output_file_directory}/"
             f"{job_name}: {e}"
         )
+
+    print(f"Saving events for '{job_name}'...")
+
     file_no = 1
-    for pv_event_stream in pv_event_streams:
-        for pv_event in pv_event_stream:
-            save_pv_event_to_file(
-                job_name, pv_event, output_file_directory, file_no
+    with tqdm(
```

Comment:
I was thinking more along the lines of having a pbar in `SQLDataHolder.stream_job_name_batches` for the number of events (if you can count rows of the table first then can get a total for the pbar) and something in `SQLDataHolder.stream_data` to tell you which job name you are streaming for.

So remove from here and add in those other places